integration-tests/basic-test-app/helpers/generateSpans.ts

```diff
@@ -0,0 +1,83 @@
+/* 'use strict'; */
+/* eslint-disable no-unused-vars */
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import {
+  context,
+  SpanKind,
+  SpanStatusCode,
+  trace,
+  Tracer,
+} from "@opentelemetry/api";
+
+export function generateTestSpans(tracer: Tracer) {
```

Comment:
Random thought: is it possible to construct a span where the end-time is before the start-time?

Reply:
that's a good point I can add an example of that to see how it's handled, I believe the interface allows that so it would be up to our implementation how it would treat that case